test/studies/shootout/reverse-complement/bradc/revcomp-blc.chpl

```diff
@@ -52,10 +48,16 @@ proc main(args: [] string) {
 }
 
 
-proc revcomp(buf: [?inds]) {
-  param cols = 61;  // the number of characters per full row (including '\n')
-  var lo = inds.low,
-      hi = inds.high;
+proc revcomp(buf, bufflo, hi) {
+  param eol = "\n".toByte(),  // end-of-line, as an integer
+        cols = 61,            // # of characters per full row (including '\n')
+        // A 'bytes' value that stores the complement of each base at its index
+        cmp = b"          \n                                                  "
+            + b"    TVGH  CD  M KN   YSAABW R       TVGH  CD  M KN   YSAABW R";
+              //    ↑↑↑↑  ↑↑  ↑ ↑↑   ↑↑↑↑↑↑ ↑       ↑↑↑↑  ↑↑  ↑ ↑↑   ↑↑↑↑↑↑ ↑
+              //    ABCDEFGHIJKLMNOPQRSTUVWXYZ      abcdefghijklmnopqrstuvwxyz
+
+  var lo = bufflo;
```

Comment:
This line is the one thing I find regrettable about this version.  Really, this should just be able to be a formal argument `in lo`, but that results in a bug in our compiler currently.  See https://github.com/chapel-lang/chapel/issues/19598  Alternatively, we could go back to the slice + bounds query approach, but that strikes me as a bit like "bundle these values up only to unbundle them within the procedure", which felt like a bit of a runaround.

Reply:
Isn't this just an `int`? Why do you need explicit `in` or the copy?

Reply:
It is, but `int` arguments are `const in` by default, and the next few lines rely on modifying this one's value, so we need to override that default or make a modifiable copy.

Reply:
Ah, gotcha.

Reply:
@e-kayrakli : Your question made me wonder "Well, why not just skip past this header outside of this routine" (which also seems appropriate since the header isn't part of the sequence to reverse).  I think the answer to this question was simply that I was trying to minimize the diff with our submitted revcomp3.chpl for the sake of comparison, but I don't think that's a good rationale in retrospect.  And that led me to do a pretty significant refactor which is even more compact and clean in #19805.